easybuild/easyconfigs/x/XALT/XALT-3.1.1.eb

```
@@ -0,0 +1,62 @@
+name = 'XALT'
+version = '3.1.1'
+
+homepage = 'https://github.com/xalt/xalt'
+description = """XALT is a tool to allow a site to track user executables
+  and library usage on a cluster. When installed it can tell a site what are
+  the top executables by Node-Hours or by the number of users or the number
+  of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+  also track package use by R, MATLAB or Python. It tracks both MPI and
+  non-MPI programs."""
+
+toolchain = SYSTEM
+
+# The location of XALT configuration file must be specified via
+# 'config_py' easyconfig parameter.  You can edit the easyconfig file,
+# or use 'eb --try-amend=config_py=<path>'. " See
+# https://xalt.readthedocs.io/en/latest/030_site_filtering.html for
+# more information.
+config_py = None
+
+# The name of the system must be specified via the 'syshost'
+# easyconfig parameter.  You can edit the easyconfig file, or use 'eb
+# --try-amend=syshost=<string>'.  See
+# https://xalt.readthedocs.io/en/latest/020_site_configuration.html
+# for more information.
+syshost = None
+
+# The XALT transmission method must be specified via the
+# 'transmission' easyconfig parameter.  You can edit the easyconfig
+# file, or use 'eb --try-amend=transmission=<string>'.  See
+# https://xalt.readthedocs.io/en/latest/020_site_configuration.html
+# for more information.
+transmission = None
+
+# Additional options
+executable_tracking = True
+gpu_tracking = None
+logging_url = None
+mysql = False
+scalar_sampling = True
+static_cxx = True
+
+source_urls = ['https://github.com/xalt/xalt/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['9c52df23c00df0bdf8af53adaf17b9ebac1b524e709cecc773431b4e88bddcbc']
+
+builddependencies = [
+    ('binutils', '2.42-GCCcore-13.3.0'),
+    ('cURL', '8.7.1-GCCcore-13.3.0'),
+    ('flex', '2.6.4-GCCcore-13.3.0'),
+    ('util-linux', '2.40-GCCcore-13.3.0'),
+    ('zstd', '1.5.6-GCCcore-13.3.0'),
```

Isn't this dependent on default MNS or something?

I would expect

```suggestion
_gcccore = {'name': 'GCCcore', 'version': '13.3.0'}

builddependencies = [
    ('binutils', '2.42', '', _gcccore),
    ('cURL', '8.7.1', '', _gcccore),
    ('flex', '2.6.4', '', _gcccore),
    ('util-linux', '2.40', '', _gcccore),
    ('zstd', '1.5.6', '', _gcccore),
```